### DISCUSSION TITLE
continued 502 bad gateway

### DISCUSSION THREAD

**Discussion by djhurlburt** [NightscoutFoundation/xDrip#3209] [Q&A] [answered]
I fell to the problem of not logging into the the freeDNS enough.  So I fixed that and then found that my status page indicated “Startup Dependence on FreeDNS” so I (re) ran Install Nightscout phase 2 and was able to get my "status" screen to look like it is supposed to (https://navid200.github.io/xDrip/docs/Nightscout/Status.html)   
![status page 11-23-2023](https://github.com/NightscoutFoundation/xDrip/assets/122819185/d432a8d5-494e-4b7a-8dca-f0a3094277a1)

I rebooted the server, but am getting "502 Bad Gateway" error, and following the instructions on this page: https://navid200.github.io/xDrip/docs/Nightscout/502BadGateway.html - after doing it a few times, it says to "contact us".  I hope I am doing that by sending this discussion.

Just in case, I just (re)ran “Install Nightscout phase 1” from the Google Cloud setup submenu and then did another reboot; verified status was still good....but am still getting the 502 bad gateway.

**Comment by Navid200** [comment, marked as answer]
Can you run phase 1 again?
Does that solve the problem?

**Reply by djhurlburt** [reply]
Hi.  I did rerun the phase 1 again, and it did not fix it <sad face>.  Wait, to be clear, do you want me to re do it?  Should I do phase 1 and then phase 2?

**Reply by Navid200** [reply]
When you log into your FreeDNS account, is the subdomain look in regular font instead of bold?
If yes, you need to create a new subdomain and use it instead of your old subdomain.  Sometimes, a subdomain is disabled and the only way to know is the font!

**Reply by djhurlburt** [reply]
it looks like it is "regular"
<deleted image>

although as I post this, the pic looks like the domain is bold, but it isn't....

**Reply by djhurlburt** [reply]
So for the heck of it, I just did phase 1 yet again, and now it is working.  The system must know you are engaged.

**Reply by Navid200** [reply]
LOL

You should delete the image that shows your hostname just to keep it confidential.   
I know that you have not disclosed your API_SECRET.
But, still. I always suggest to everyone to also keep their hostname private.

I am pleased it is working again.
